resources/waf_v2_rule_groups.go

```diff
@@ -0,0 +1,78 @@
+package resources
+
+import (
+	"context"
+
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/service/wafv2"
+	"github.com/cloudquery/cq-provider-aws/client"
+	"github.com/cloudquery/cq-provider-sdk/provider/schema"
+)
+
+func WafV2RuleGroups() *schema.Table {
```

Comment:
missing [GetPermissionPolicy ](https://docs.aws.amazon.com/waf/latest/APIReference/API_GetPermissionPolicy.html)for the RuleGroup add this a json policy column

Reply:
Thanks!